clueless/views.py

```diff
@@ -60,3 +146,192 @@ def playerturn(request):
 	return HttpResponse(template.render(context,request))
 
 # Controller functions will go below here
+@login_required
+def start_game_controller(request):
+	"""
+	Creates a game with the given host defined by his/her user_id. Also provided
+	is the host's designated character. Other players may join later, as this
+	game is not available in the lobby as a joinable game.
+	"""
+	if request.method == 'POST':
+		if 'character_id' not in request.POST:
+			logger.error('character_id not provided')
+			return redirect('startgame')
+		else:
+			#can get logged in user direct from request object
+			user = request.user
+			character_id = request.POST.get('character_id')
+
+			try:
+				character = Character.objects.get(card_id = character_id)
+			except ObjectDoesNotExist: # Possible User.DoesNotExist
+				logger.error('''character not found (Did you forget to add the
+				character in the admin panel?''')
+				return redirect('startgame')
+
+			# Constructs our game, saves the changes and starts it
+			game = Game()
+
+			# Create a Player object for the Host
+			player = Player()
+			player.user = user
+			player.character = character
+			player.currentSpace = character.defaultSpace
+			player.save()
+
+			game.initializeGame(player)
+			game.save()
+
+			game.addPlayer(player)
+			game.save()
+
+			# kewl, we are done now.  Let's send our user to the game interface
+			return redirect('playgame', game_id = game.id)
```

Comment:
Hmm I commented on the other pull request about this logic as well, but I think this isn't quite what we want either because this just initializes the game and then plays it? Need to think about this more..